### PR TITLE
Update ElysiumCharacterTable.kt

### PR DESCRIPTION
**Pull request by balohmatevz**

### CODE REVIEW

File: bukkit/elysium-characters-bukkit/src/main/kotlin/com/seventh_root/elysium/characters/bukkit/database/table/ElysiumCharacterTable.kt [hunk 1]
```diff
@@ -114,7 +114,7 @@ class ElysiumCharacterTable: Table<ElysiumCharacter> {
             exception.printStackTrace()
         }
 
-        if (database.getTableVersion(this) == null) {
+        if (database.getTableVersion(this) == null) {
```
Comment (alyphen) [inline bukkit/elysium-characters-bukkit/src/main/kotlin/com/seventh_root/elysium/characters/bukkit/database/table/ElysiumCharacterTable.kt]: Please fix trailing whitespace.

Comment (balohmatevz) [inline bukkit/elysium-characters-bukkit/src/main/kotlin/com/seventh_root/elysium/characters/bukkit/database/table/ElysiumCharacterTable.kt]: No, Hinamizawa!

**Comment by alyphen** [comment]
Issues with this changeset have not been solved over several version releases. Functionality change is not significant enough to keep this open.
Closing this, if you wish to update this PR to the current version and fix remaining issues please re-open.

**Comment by balohmatevz** [comment]
REEEEE

**Pull request closed without merging**